ee/codegen/src/__test__/nodes/__snapshots__/inline-prompt-node.test.ts.snap

```diff
@@ -1297,3 +1297,32 @@ class PromptNodeDisplay(BaseInlinePromptNodeDisplay[PromptNode]):
     )
 "
 `;
+
+exports[`InlinePromptNode > with ml model node attribute defined > getNodeFile 1`] = `
+"from vellum import JinjaPromptBlock, PromptParameters
+from vellum.workflows.nodes.displayable import InlinePromptNode
+
+from ..inputs import Inputs
+
+
+class PromptNode(InlinePromptNode):
+    ml_model = "gpt-4"
+    blocks = [
+        JinjaPromptBlock(template="""Summarize what this means {{ INPUT_VARIABLE }}"""),
+    ]
+    prompt_inputs = {
+        "text": Inputs.text,
+    }
```

Comment:
[oos] we should clean up these tests eventually, such that these don't generate in the default factory case, and that way we could focus each of these tests on the incremental functionality added